pathd/path_pcep_controller.c

```diff
@@ -753,7 +757,43 @@ int pcep_main_event_handler(struct thread *thread)
 
 
 /* ------------ Helper functions ------------ */
+int is_best_pce(struct ctrl_state *ctrl_state, int pce)
```

Comment:
Instead of doing that, the controller should elect a pcc every time a new one appears or one of them fail to connect for a maximum number or retries/time, and update them by calling some pcc api function, like the last comment.

Reply:
same al above